docs/user-guide/getting-started.md

```diff
@@ -1,182 +1,188 @@
+3. Now, create a `Mapping` configuration that tells Ambassador to route all traffic from `/backend/` to the `quote` service. Copy the following YAML and save it to a file called `quote-backend.yaml`.
 
-   ```yaml
-   ---
-   apiVersion: getambassador.io/v2
-   kind: Mapping
-   metadata:
-     name: quote-backend
-     namespace: ambassador
-   spec:
-     prefix: /backend/
-     service: quote
-   ```
+  ```yaml
+  ---
+  apiVersion: getambassador.io/v2
+  kind: Mapping
+  metadata:
+    name: quote-backend
+    namespace: ambassador
+  spec:
+    prefix: /backend/
+    service: quote
+  ```
 
 4. Apply the configuration to the cluster by typing the command `kubectl apply -f quote-backend.yaml`.
 
 5. Test the configuration by typing `curl -Lk https://<hostname>/backend/` or `curl -Lk https://<IP address>/backend/`. You should see something similar to the following:
 
-   ```
-   (⎈ |rdl-1:default)$ curl -Lk https://aes.ri.k36.net/backend/
-   {
-    "server": "idle-cranberry-8tbb6iks",
-    "quote": "Non-locality is the driver of truth. By summoning, we vibrate.",
-    "time": "2019-12-11T20:10:16.525471212Z"
-   }
-   ```
+  ```
+  (⎈ |rdl-1:default)$ curl -Lk https://aes.ri.k36.net/backend/
+  {
+   "server": "idle-cranberry-8tbb6iks",
+   "quote": "Non-locality is the driver of truth. By summoning, we vibrate.",
+   "time": "2019-12-11T20:10:16.525471212Z"
+  }
+  ```
+
+## A Single Source of Configuration
 
-## A single source of configuration
+In the Ambassador Edge Stack, Kubernetes serves as the single source of configuration. Changes made on the command line (via `kubectl`) are reflected in the Edge Policy Console, and vice versa. This enables a consistent configuration workflow.
 
-1. In the Ambassador Edge Stack, Kubernetes serves as the single source of configuration. Changes made on the command line (via `kubectl`) are reflected in the UI, and vice versa. This enables a consistent configuration workflow. You can see this in action by navigating to the Mappings tab. You'll see an entry for the `quote-backend` Mapping that was just created on the command line.
+1. To see this in action, navigate to the **Mappings** tab. You'll see an entry for the `quote-backend` Mapping that was just created on the command line.
 
-2. If you configured TLS, you can type `kubectl get hosts` to see the `Host` resource that was created:
+2. Type `kubectl get hosts` to see the `Host` resource that was created:
 
-   ```
-   (⎈ |rdl-1:default)$ kubectl get hosts
-   NAME               HOSTNAME           STATE   PHASE COMPLETED   PHASE PENDING   AGE
-   aes.ri.k36.net     aes.ri.k36.net     Ready                                    158m
-   ```
+```  
+  (⎈ |rdl-1:default)$ kubectl get hosts
+  NAME               HOSTNAME           STATE   PHASE COMPLETED   PHASE PENDING   AGE
+ blackbird-penguin-123.edgestack.me      blackbird-penguin-123.edgestack.me      Ready                                    158m
```

Comment:
😂 cool name!
Please make sure NAME, HOSTNAME and STATE align with the content (table-like)